spec/features/admin/dashboard_spec.rb

```diff
@@ -1,15 +1,49 @@
 require "rails_helper"
 
 describe "admin dashboard" do
+  let!(:admin) { create :admin }
+
+  context "authorization" do
+    xit 'only admins can see the admin dashboard' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.